showkase-annotation/src/main/java/com/airbnb/android/showkase/annotation/ShowkaseComposable.kt

```diff
@@ -47,8 +47,8 @@ package com.airbnb.android.showkase.annotation
  * Use this to restrict the size of your preview inside the Showkase browser.
  */
 @MustBeDocumented
-@Retention(AnnotationRetention.RUNTIME)
-@Target(AnnotationTarget.FUNCTION, AnnotationTarget.TYPE)
```

Comment:
Trying to think if I added `TYPE` intentionally 🤔

Reply:
the processor in its current form would fail if any classes were annotated, so it shouldnt be there now at least